### DISCUSSION TITLE
could not resolve proxy: proxy.server.com'

### DISCUSSION THREAD

**Discussion by shrushti4567** [community/community#36374] [Other Feature Feedback, Questions, & Ideas]
whenever i git clone, git push  my terminal show the the the error 'could not resolve proxy: proxy.server.com'

**Comment by airtower-luna** [comment]
That sounds like your [Git configuration](https://git-scm.com/docs/git-config) sets a proxy, but your computer then can't find the IP address for the hostname. What to do depends on whether you actually want/need to use the proxy:

* If you don't want to use the proxy, remove the proxy settings from your Git config, so Git won't try to use the proxy any more.
* If you want to use the proxy you'll have to investigate *why* your computer can't find the proxy. If it's a corporate/university/etc. proxy, you might want to talk to an admin.

**Comment by ossanna16** [comment]
Hey there @shrushti4567 and thank you for asking a good question! I'm glad you found our community 🙂

**Comment by mohitkg01** [comment]
can anyone share how we can solve it
